ci/nightly.sh

```diff
@@ -38,9 +38,10 @@ build_appimage() {
   (
     require_environment_variable NEOVIM_DIR "${BASH_SOURCE[0]}" ${LINENO}
 
-    sudo modprobe fuse;
-    user="$(whoami)";
-    sudo usermod -a -G fuse $user;
+    sudo modprobe fuse
+
+    sudo groupadd fuse
```

Comment:
if the group doesn't exist, maybe it's not necessary anymore? This was previously needed so that the current user was allowed to use FUSE. Seems unlikely that adding the user to a group that we just created will have any effect ?

Reply:
I followed these steps from https://github.com/AppImage/AppImageKit/wiki/FUSE#install-fuse